### PR TITLE
Addition of copyright model and related fixtures

### PR DESCRIPTION
**Pull request by kjallen**
Addition of Copyright models and partial fixtures for each table.

To confirm addition and data:

Restart application or run django migration process
docker-compose exec django python manage.py migrate
Load related data:

docker-compose exec django python manage.py loaddata copyright-type-data
docker-compose exec django python manage.py loaddata copyright-data

Navigate to the [django admin interface](http://localhost:8000/admin/) and confirm models exist and data is visible.

### CODE REVIEW

**Review by akohler — CHANGES_REQUESTED** [review]
@kjallen This looks OK to me, but please add the two `loaddata` commands to `entrypoint.sh`, following https://github.com/UCLALibrary/oral-history-staff-ui/blob/SYS-1175/copyright_models_and_data/docker_scripts/entrypoint.sh#L36

That ensures the sample data is consistently loaded on startup (for developers).  Once real data is available, we may switch to those lists.

Thanks --Andy

**Comment by kjallen** [comment]
Yes, thank you for reminder @akohler, I had forgotten that

File: oh_staff_ui/models.py [hunk 1]
```diff
@@ -258,3 +258,32 @@ class ItemPublisherUsage(models.Model):
...
+
+    def __str__(self):
+        return self.value
+
+
+class ItemCopyrightUsage(models.Model):
+    item = models.ForeignKey(
+        ProjectItem, on_delete=models.PROTECT, blank=False, null=False
+    )
+    publisher = models.ForeignKey(
```
Comment (akohler) [inline oh_staff_ui/models.py:284]: @kjallen Didn't notice this before, sorry, but `publisher` here seems wrong.  Should this be `copyright` or `copyright_statement` or something else?

**Review by akohler — CHANGES_REQUESTED** [review]

File: oh_staff_ui/models.py (hunk 1 above)
Comment (kjallen) [reply oh_staff_ui/models.py]: Thank you for catching @akohler, corrected and updated.

**Review by akohler — APPROVED** [review]

**Pull request merged**